pkg/descheduler/evictions/evictions.go

```diff
@@ -561,6 +563,24 @@ func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, opts EvictOptio
 	return nil
 }
 
+func (pe *PodEvictor) simulateSchedulingInDryRun(pod *v1.Pod) {
```

Comment:
I don't quite understand why we need to create a pod in dry run mode. The purpose of dry run mode is to simulate the behavior of Descheduler to preview which pods may be evicted. 🤔

Reply:
see comments：https://github.com/kubernetes-sigs/descheduler/issues/1584#issuecomment-2564966754

> I don't quite understand why we need to create a pod in dry run mode. 

The purpose is to reserve the resources.

If any part is not described clearly, please feel free to comment.

Reply:
got this. thanks